department-of-veterans-affairs/va.gov-cms
# VACMS-6088: Add buttons for service location deletion.

## Description
Closes #6088

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/129071008-54985c3a-f53b-45c3-b03e-5713943ee311.png)
![image](https://user-images.githubusercontent.com/2404547/129071030-f97ef525-d6f1-4001-86d1-e2c3382fecb8.png)

## QA steps
- Visit node/4680/edit and visually verify that locations have three dot menus over each instance, containing delete options.
- Delete one, and visually verify three dot menu does not appear over single instance.

### Definition of Done

### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
